src/control/server/ctl_storage_rpc.go

```diff
@@ -71,7 +84,7 @@ func scmModulesToPB(mms []storage.ScmModule) (pbMms pb_types.ScmModules) {
 	return
 }
 
-func scmNamespacesToPB(nss []storage.ScmNamespace) (pbNss pb_types.ScmNamespaces) {
+func scmNamespacesToPB(nss []storage.ScmNamespace) (pbNss proto.ScmNamespaces) {
```

Comment:
implement the from/toNative methods as you did above for nvme controllers?

Reply:
Yeah, eventually. My plan was to focus on NVMe for this PR so that I can regain functional parity with master and then take another pass through to refactor after I finish the privileged binary work.